jest.config.ts

```diff
@@ -9,6 +12,9 @@ const createJestConfig = nextJest({
 // Add any custom config to be passed to Jest
 const config: Config = {
   coverageProvider: 'v8',
+  moduleNameMapper: {
```

Comment:
This was added because when I writing down the path `jest.mock('@utils/utils')`, it didn't recognize the file path and I had to use `jest.mock('../../../utils/utils')`.